Replicator/IncomingRev.cc

```diff
@@ -176,7 +176,8 @@ namespace litecore { namespace repl {
             // have properties to decrypt.
             logVerbose("Need to apply delta immediately for '%.*s' #%.*s ...",
                        SPLAT(_rev->docID), SPLAT(_rev->revID));
-            fleeceDoc = _db->applyDelta(_rev->docID, _rev->deltaSrcRevID, jsonBody);
+            fleeceDoc = _db->applyDelta(replicator()->collection(collectionIndex()),
```

Comment:
The whole get the replicator then get the collection from the collection index could stand its own method on the workers.  Readability wise it's three collection calls crammed together in one line.

Reply:
yes, accepted. Will change